examples/bokeh_handler.py

```diff
@@ -0,0 +1,25 @@
+import logging
```

Comment:
This file should not be included, as this functionality exists in bokeh_garden/logging_handler.py above